src/components/Story/Story.js

```diff
@@ -3,9 +3,11 @@ import "./Story.css";
 import Day from "../Day/Day";
 import Choices from "../Choices/Choices";
 import Result from "../Result/Result";
-import { Switch, Route } from "react-router-dom";
+import { Switch, Route, NavLink } from "react-router-dom";
+import { useSelector } from "react-redux";
 
 function Story() {
+  const dayNum = useSelector((state) => state.day.value);
```

Comment:
How does it feel using redux to access this data in several components compared to having to pass it through as props?